src/DynamoCoreWpf/Utilities/ResourceUtilities.cs

```diff
@@ -601,13 +602,21 @@ internal static async void LoadWebBrowser(HtmlPage htmlPage, WebView2 webBrowser
                     };
                 }
 
-                await webBrowserComponent.EnsureCoreWebView2Async();
-                // Context menu disabled
-                webBrowserComponent.CoreWebView2.Settings.AreDefaultContextMenusEnabled = false;
-                webBrowserComponent.NavigateToString(bodyHtmlPage);
+                try
+                {
+                    await webBrowserComponent.Initialize();
+
+                    // Context menu disabled
+                    webBrowserComponent.CoreWebView2.Settings.AreDefaultContextMenusEnabled = false;
+                    webBrowserComponent.NavigateToString(bodyHtmlPage);
 
-                webBrowserComponent.CoreWebView2.Settings.IsZoomControlEnabled = false;
-                webBrowserComponent.CoreWebView2.Settings.IsPinchZoomEnabled = false;
+                    webBrowserComponent.CoreWebView2.Settings.IsZoomControlEnabled = false;
+                    webBrowserComponent.CoreWebView2.Settings.IsPinchZoomEnabled = false;
+                }
+                catch (ObjectDisposedException ex)
+                {
+                    Debug.WriteLine(ex.Message);
```

Comment:
only log in debug? We don't want to note something the dynamo console somehow?

Reply:
added console.writeLine